
### Examining racial discrimination in the US job market

#### Background
Racial discrimination continues to be pervasive in cultures throughout the world. Researchers examined the level of racial discrimination in the United States labor market by randomly assigning identical résumés black-sounding or white-sounding names and observing the impact on requests for interviews from employers.

#### Data
In the dataset provided, each row represents a resume. The 'race' column has two values, 'b' and 'w', indicating black-sounding and white-sounding. The column 'call' has two values, 1 and 0, indicating whether the resume received a call from employers or not.

Note that the 'b' and 'w' values in race are assigned randomly to the resumes.

#### Exercise
You will perform a statistical analysis to establish whether race has a significant impact on the rate of callbacks for resumes.

Answer the following questions **in this notebook below and submit to your Github account**. 

   1. What test is appropriate for this problem? Does CLT apply?
   2. What are the null and alternate hypotheses?
   3. Compute margin of error, confidence interval, and p-value.
   4. Discuss statistical significance.

You can include written notes in notebook cells using Markdown: 
   - In the control panel at the top, choose Cell > Cell Type > Markdown
   - Markdown syntax: http://nestacms.com/docs/creating-content/markdown-cheat-sheet


#### Resources
+ Experiment information and data source: http://www.povertyactionlab.org/evaluation/discrimination-job-market-united-states
+ Scipy statistical methods: http://docs.scipy.org/doc/scipy/reference/stats.html 
+ Markdown syntax: http://nestacms.com/docs/creating-content/markdown-cheat-sheet

****

In [1]:
import pandas as pd
import numpy as np
from scipy import stats

In [2]:
data = pd.io.stata.read_stata('us_job_market_discrimination.dta')

In [3]:
# number of callbacks for balck-sounding names
sum(data[data.race=='b'].call)

157.0

In [8]:
data[data.race=='b'].call.describe()

count    2435.000000
mean        0.064476
std         0.245649
min         0.000000
25%         0.000000
50%         0.000000
75%         0.000000
max         1.000000
Name: call, dtype: float64

In [9]:
data[data.race=='w'].call.describe()

count    2435.000000
mean        0.096509
std         0.295346
min         0.000000
25%         0.000000
50%         0.000000
75%         0.000000
max         1.000000
Name: call, dtype: float64

####  Is the callback rate significantly different for blacks and whites?

1. The Central Limit Theorem aplies to any distribution with a large enough sample. Here the sample is 4870, with 50% black and 50% white. Both subsamples are large enough. The callback rates are  0.064476 and 0.096509, as can be seen above.

2. The appropriate test is a comparison of means between two distributions. The null hypothesis is that the means are equal. The alternative hypothesis is that the means are different.

3. a) Margin of error = z*sigma/sqrt(n)

   b) Confidence interval = [mean - z*sigma/sqrt(n), mean + z*sigma/sqrt(n)]

In [11]:
aa=data[data.race=='b'].call.as_matrix()
bb=data[data.race=='w'].call.as_matrix()
stats.ttest_ind(aa,bb)


Ttest_indResult(statistic=-4.1147052908617514, pvalue=3.9408021031288859e-05)

####  Is there significant racial discrimination?

The callback rate for blacks, about 2/3 that for whites, is very significantly lower. A p value of 0.00004 is very significant indeed.
